CameraInterface/CameraInterface.py

```diff
@@ -0,0 +1,64 @@
+##
+# @file CameraInterface.py
+# @brief カメラから画像を取得、保存を行う
+# @author kawano
+# 実行コマンド PowerShell.exe -ExecutionPolicy RemoteSigned -File captureCameraInterface.ps1 カメラ番号
+import os
+# cv2.VideoCaptureの処理時間短縮(import cv2の前に書く必要あり)
+# @see https://qiita.com/youichi_io/items/b894b85d790720ea2346
+os.environ["OPENCV_VIDEOIO_MSMF_ENABLE_HW_TRANSFORMS"] = "0"
+import cv2
+import sys
+
+# カメラ仲介クラス
+class CameraInterface:
+    ##
+    # @brief コンストラクタ
+    # @param cameraId カメラ番号
+    def __init__(self, cameraId):
```

Comment:
このままでは、引数の型が分からないのでType Hintsを導入していただけると助かります。
また、上記のコメントで述べたように関数の説明にはDocstringを用いていただけると幸いです。

```suggestion
    def __init__(self, cameraId: int) -> None:
        """コンストラクタ.

        Args:
            cameraId (int): カメラ番号
        """
```